scripts/create-yearly-es-globals.mjs

```diff
@@ -0,0 +1,21 @@
+import fs from 'node:fs';
+
+const year = new Date().getFullYear();
+const dataFile = new URL(`../data/es${year + 1}.mjs`, import.meta.url);
+const workflowFile = new URL('../.github/workflows/es-yearly-globals.yml', import.meta.url);
+
+fs.writeFileSync(
+	dataFile,
+	`export {default} from './es${year}.mjs';\n`,
+);
+
+fs.writeFileSync(
+	workflowFile,
+	fs.readFileSync(workflowFile, 'utf8')
+		.replaceAll(
+			`title: Add \`es${year + 1}\` globals`,
+			`title: Add \`es${year + 2}\` globals`,
+		),
```

Comment:
This will not match `commit-message:` from the workflow file.

Reply:
You are looking at the first commit, the sencond commit fixed it.

Reply:
Sorry. I should not review PRs on the GitHub app. It makes it too hard to see things like this.